package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-provisioned-concurrency-autoscaling",
-  "version": "1.9.1",
+  "version": "2.0.0",
```

Comment:
I think will make a breaking change. Last time I tried it we had to roll back. Probably best to go with 1.10.0.

Reply:
@claydanford There is a breaking change with this PR. The serverless 3 update resulted in change in schema since pc attribute is now a default thats defined in serverless3 and also role creation standards have changed